frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/storage-efficiency-card/storage-efficiency-card-item.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { FieldLevelHelp, humanizeBinaryBytes } from '@console/internal/components/utils';
+
+const ItemBody: React.FC<ItemBodyProps> = React.memo(
+  ({ title, stats, infoText, isLoading, error }) => {
+    let status: React.ReactElement;
+    if (isLoading) {
+      status = <div className="skeleton-text ceph-storage-efficiency-card__item-body--loading" />;
+    } else if (error || !stats) {
+      status = <span className="co-dashboard-text--small text-muted">Not available</span>;
+    } else {
+      status = <span className="ceph-storage-efficiency-card__row-status-item-text">{stats}</span>;
+    }
+    return (
+      <div className="co-inventory-card__item">
+        <div className="ceph-storage-efficiency-card__row-title">{title}</div>
+        <div className="ceph-storage-efficiency-card__row-status-item">
```

Comment:
Since these are rows for the `ItemBody`
```suggestion
      <div className="co-inventory-card__item-row">
        <div className="ceph-storage-efficiency-card__item-row-title">{title}</div>
        <div className="ceph-storage-efficiency-card__item-row-status">
```

Reply:
Also `ItemBody` can be `RowItem`

Reply:
We have to keep `co-inventory-card__row` class as it is becasue it is used across dashboard https://github.com/openshift/console/blob/master/frontend/packages/console-shared/src/components/dashboard/inventory-card/inventory-card.scss#L4.

Reply:
I dont find this naming right though.